core/src/main/java/org/apache/accumulo/fate/Fate.java

```diff
@@ -300,6 +328,56 @@ public TStatus waitForCompletion(long tid) {
     return store.waitForStatusChange(tid, FINISHED_STATES);
   }
 
+  /**
+   * Attempts to cancel a running Fate transaction
+   *
+   * @param tid
+   *          transaction id
+   * @return true if transaction transitioned to a failed state or already in a completed state,
+   *         false otherwise
+   */
+  public boolean cancel(long tid) {
+    String tidStr = Long.toHexString(tid);
+    for (int retries = 0; retries < 5; retries++) {
+      if (store.tryReserve(tid)) {
+        try {
+          TStatus status = store.getStatus(tid);
+          if (status == TStatus.NEW || status == TStatus.SUBMITTED
+              || status == TStatus.IN_PROGRESS) {
+            store.setProperty(tid, EXCEPTION_PROP, new TApplicationException(
```

Comment:
What is `EXCEPTION_PROP`? I can't seem to find it.

Reply:
See https://github.com/apache/accumulo/blob/main/core/src/main/java/org/apache/accumulo/fate/Fate.java#L48